lib/global.gi

```diff
@@ -97,6 +97,21 @@ InstallGlobalFunction( IsBoundGlobal,
 end);
 
 
+#############################################################################
+##
+#M  IsAutoGlobal ( <name> )  . . . .  check if a global is automatic
+## 
+
+
+InstallGlobalFunction( IsAutoGlobal, 
+        function (name)
+    local isauto;
+    CheckGlobalName( name );
+    isauto := IS_AUTO_GVAR(name);
+    Info( InfoGlobal, 3, 
+          "IsAutoGlobal: called for ", name, " returned ", isauto);
```

Comment:
What's the motivation for this `Info`? Only to imitate what's done elsewhere in this file? Is there any scenario this might be useful?

Reply:
Yes, for consistency with IsBoundGlobal, and others. we could consistently simplify this whole file of course.